content/guides/translation-connection/step3-interact-with-julia.md

```diff
@@ -0,0 +1,241 @@
+# Interact with Julia
```

Comment:
I don't like this title as it feels a bit blurry. By reading this title, the user won't know what's inside this part. As you describe here the whole "How to" regarding the connector building, I would call it this way "How to build your connector?"

Reply:
Good idea.